### DISCUSSION TITLE
Dashboards does not load properly.

### DISCUSSION THREAD

**Discussion by vladimir-remar** [apache/superset#16315] [Q&A / Help]
### Expected results

A dashboard properly loaded before its sended in email

### Actual results.
It seems even with EMAIL_PAGE_RENDER_WAIT increased does not wait the time to take the screenshot
```
Aug 13 11:42:00 ip-172-31-47-99 bash[170558]: Scheduling alert email_test eta: 2021-08-13 11:42:00+02:00
Aug 13 11:42:00 ip-172-31-47-99 bash[170558]: Init selenium driver
Aug 13 11:42:07 ip-172-31-47-99 bash[170558]: Taking a PNG screenshot or url https://XXXXXX/superset/dashboard/14/
Aug 13 11:42:11 ip-172-31-47-99 bash[170558]: Report sent to email
```

#### Screenshots
What I get
![bad_results](https://user-images.githubusercontent.com/22233596/129347550-4a20f661-b217-4c09-a132-d843eb76aefd.png)

What I Expect
![god_results](https://user-images.githubusercontent.com/22233596/129347405-da479722-0882-4b87-8b8c-b4f22a498aae.jpg)

#### How to reproduce
I create a properly enviroment for superset and run the following commands:

```
export SUPERSET_CONFIG_PATH=/home/ubuntu/Work/Superset/superset_config.py
gunicorn -w 10 -k gevent -b  0.0.0.0:8080 --limit-request-line 0 --limit-request-field_size 0 --log-file=/home/ubuntu/supersetDir/gunicorn.log "superset.app:create_app()"
celery worker --app=superset.tasks.celery_app:app --pool=prefork -O fair -c 4 --logfile=/home/ubuntu/supersetDir/celeryworker.logs --pidfile=/home/ubuntu/supersetDir/celery-worker.pid --loglevel=info
celery beat --app=superset.tasks.celery_app:app --logfile=/home/ubuntu/supersetDir/celerybeatlog.logs
```
### Environment

- superset version: 1.2
- python version: Python 3.8.10
- any feature flags active:
 
```
FEATURE_FLAGS = {
    "ALERT_REPORTS": True,
    "DASHBOARD_NATIVE_FILTERS": True,
    "THUMBNAILS": True,
    "OMNIBAR": True,
    "LISTVIEWS_DEFAULT_CARD_VIEW": True,
    "THUMBNAILS_SQLA_LISTENERS": True,
}
```

Part of my superset_config.py

```
class CeleryConfig(object):
    BROKER_URL = 'redis://%s:%s/0' % (REDIS_HOST, REDIS_PORT)
    CELERY_IMPORTS = ('superset.sql_lab', "superset.tasks", "superset.tasks.thumbnails", )
    CELERY_RESULT_BACKEND = 'redis://%s:%s/0' % (REDIS_HOST, REDIS_PORT)
    CELERYD_PREFETCH_MULTIPLIER = 10
    CELERY_ACKS_LATE = True
    CELERY_ANNOTATIONS = {
        'sql_lab.get_sql_results': {
            'rate_limit': '100/s',
        },
        'email_reports.send': {
            'rate_limit': '1/s',
            'time_limit': 600,
            'soft_time_limit': 600,
            'ignore_result': True,
        },
    }
    CELERYBEAT_SCHEDULE = {
        'email_reports.schedule_hourly': {
            'task': 'email_reports.schedule_hourly',
            'schedule': crontab(minute=1, hour='*'),
        },
        'alerts.schedule_check': {
            'task': 'alerts.schedule_check',
            'schedule': crontab(minute='*', hour='*'),
        },
        'reports.scheduler': {
            'task': 'reports.scheduler',
            'schedule': crontab(minute='*', hour='*'),
        },
        'reports.prune_log': {
            'task': 'reports.prune_log',
            'schedule': crontab(minute=0, hour=0),
        },
        'cache-warmup-hourly': {
            'task': 'cache-warmup',
            'schedule': crontab(minute='*/30', hour='*'),
             'kwargs': {
                'strategy_name': 'top_n_dashboards',
                'top_n': 10,
                'since': '7 days ago',
            },
       },
    }

CELERY_CONFIG = CeleryConfig

#Config Email notfication
EMAIL_REPORTS_CRON_RESOLUTION = 15
EMAIL_PAGE_RENDER_WAIT = 30
EMAIL_NOTIFICATIONS = True
EMAIL_PAGE_RENDER_WAIT = 30
```
### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by villebro** [comment]
Hmm, it appears we should introduce a chart prop to skip animations that is set to `true` when thumbs/reports are rendered.

**Comment by vladimir-remar** [comment]
@villebro Thanks for answer, Now I get it, any advice on this? or just wait to the implementation?

**Comment by villebro** [comment]
My guess is this would require the following:
- a PR on https://github.com/apache-superset/superset-ui that adds a `disableAnimations` prop to the `ChartProps` class.
- Add support for a `disableAnimations` URL parameter that then gets passed to the chart component.

It shouldn't be that hard to do if you know TypeScript and React. Let me know if you want to take this on!

**Comment by nytai** [comment]
You can also configure 
https://github.com/apache/superset/blob/6cd15d54a0ca503b6d050e296d66876383c31365/superset/config.py#L507
